example/kustomization.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
   # exclude cluster resources for a namespaced install
-  #- github.com/utilitywarehouse/argocd-manifests/cluster?ref=1.6.1-3
+  #- github.com/utilitywarehouse/argocd-manifests/cluster?ref=1.6.2
   - ../cluster
-  #- github.com/utilitywarehouse/argocd-manifests/namespaced?ref=1.6.1-3
+  #- github.com/utilitywarehouse/argocd-manifests/namespaced?ref=1.6.2
```

Comment:
The next version would be `1.6.2-1`. To be honest we should just replace the version reference with a placeholder like `github.com/utilitywarehouse/argocd-manifests/cluster?ref=<version>` so we don't need to keep updating it all the time.

Reply:
Corrected